hawkmoth/parser.py

```diff
@@ -316,7 +388,7 @@ def _clang_diagnostics(diagnostics, errors):
                                   diag.location.line, diag.spelling))
 
 # Parse a file and return a tree of docstring.Docstring objects.
-def parse(filename, clang_args=None):
+def parse(filename, domain='c', clang_args=None):
```

Comment:
I realize that "domain" is a Sphinx thing, and this sort of "leaks" that into the parser, which I try to avoid.

I'm going to ignore it though, because it's not that bad in this case, and I don't want to force another respin. And I don't even know yet what it would look like, and it's easier to fix afterwards, if indeed it needs to be fixed at all.

I'm thinking maybe it should just be "language" in the parser level, and maybe it could be an enum. And then it would be translated to domain in docstring. Anyway, not worth it at this time, just musing.

Reply:
Good point. Will keep in mind :)